crates/propolis-api-types/src/lib.rs

```diff
@@ -186,6 +194,13 @@ pub struct InstanceProperties {
     pub vcpus: u8,
 }
 
+impl InstanceProperties {
+    /// Return the name of the VM as managed by the VMM.
```

Comment:
Perhaps clearer as "Name of the VMM resource backing this VM"?

Reply:
Reworded in d61fd69